deployment/deploy/3000-deploy-VestingRegistry.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+const { getContractNameFromScriptFileName } = require("../helpers/utils");
+const { sendWithMultisig } = require("../helpers/helpers");
+const { deployWithCustomProxy } = require("../helpers/helpers");
+const col = require("cli-color");
+//const deploymentName = getContractNameFromScriptFileName(path.basename(__filename));
+const func = async function (hre) {
+    const {
+        deployments: { deploy, get, log },
+        getNamedAccounts,
+        ethers,
+    } = hre;
+    const { deployer } = await getNamedAccounts();
+    log(col.bgYellow("Deploying VestingRegistry..."));
+    await deployWithCustomProxy(
+        deployer,
+        "VestingRegistryLogic",
+        "VestingRegistryProxy",
+        "",
+        "VestingRegistry_Proxy",
+        true
+    );
```

Comment:
There is a confusion because we have VestingRegistry contract which was merged into a single vesting registry together with VestingRegistry2 and VestignRegistry3 - all of those are deprecated.    
I suggest the following clean-up for clarity:
1) remove legacy vesting registry contracts VestingRegistry, VestingRegistry2, VestingRegistry3
2) rename VestingRegistryLogic to VestingRegistry
3) update tests and deployment naming  

```suggestion
    await deployWithCustomProxy(
        deployer,
        "VestingRegistry",
        "VestingRegistryProxy",
        "",
        "VestingRegistry_Proxy",
        true
    );
```